src/main/java/Command/Command.java

```diff
@@ -0,0 +1,34 @@
+package Command;
+
+import java.time.LocalDateTime;
+
+import Errors.ErrorExceptions;
+
+/**
+ * Represents the overall command abstract class for sub-class to follow.
+ */
+public abstract class Command {
+
+    /**
+     * Sets the method to be followed by sub-class.
+     * @param s String.
+     */
+    public static void execute(String s) throws ErrorExceptions {}
+
+    /**
+     * Sets the method to be followed by sub-class.
+     */
+    public static void execute() {}
+
+    /**
+     * Sets the method to be followed by sub-class.
+     * @param i integer.
+     */
+    public static void execute(int i) throws ErrorExceptions {}
```

Comment:
Hey, the variable names for the parameters along with the javadoc comments don't seem to give much information regarding what the function expects. I see that the classes extending this one have more specific comments and variable names which is great! Perhaps this could be made more descriptive?